k8s/plugin/README.md

```diff
@@ -28,39 +62,39 @@ The plugin needs to be able to connect to the REST server in order to make the a
 1. Get Volumes
 ```
 ❯ kubectl mayastor get volumes
- ID                                    REPLICAS TARGET-NODE  ACCESSIBILITY STATUS  SIZE
- 18e30e83-b106-4e0d-9fb6-2b04e761e18a  4        mayastor-1   nvmf          Online  10485761
- 0c08667c-8b59-4d11-9192-b54e27e0ce0f  4        mayastor-2   <none>        Online  10485761
+ ID                                    REPLICAS TARGET-NODE  ACCESSIBILITY STATUS  SIZE     THIN-PROVISIONED ALLOCATED
+ 18e30e83-b106-4e0d-9fb6-2b04e761e18a  4        kworker1     nvmf          Online  1GiB     true             8MiB
+ 0c08667c-8b59-4d11-9192-b54e27e0ce0f  4        kworker2     <none>        Online  381.6MiB false            384MiB
 
 ```
 2. Get Volume by ID
 ```
 ❯ kubectl mayastor get volume 18e30e83-b106-4e0d-9fb6-2b04e761e18a
- ID                                    REPLICAS TARGET-NODE  ACCESSIBILITY STATUS  SIZE
- 18e30e83-b106-4e0d-9fb6-2b04e761e18a  4        mayastor-1   nvmf          Online  10485761
+ ID                                    REPLICAS TARGET-NODE  ACCESSIBILITY STATUS  SIZE     THIN-PROVISIONED ALLOCATED
+ 18e30e83-b106-4e0d-9fb6-2b04e761e18a  4        kworker1     nvmf          Online  1GiB     true             8MiB
 
 ```
 3. Get Pools
 ```
 ❯ kubectl mayastor get pools
- ID               TOTAL CAPACITY  USED CAPACITY  DISKS                                                     NODE      STATUS  MANAGED
- mayastor-pool-1  5360320512      1111490560     aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  kworker1  Online  true
- mayastor-pool-2  5360320512      2172649472     aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  kworker1  Online  true
- mayastor-pool-3  5360320512      3258974208     aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  kworker2  Online  false
+ ID               DISKS                                                     MANAGED  NODE      STATUS  CAPACITY  ALLOCATED  AVAILABLE
+ pool-1-kworker1  aio:///dev/vdb?uuid=d8a36b4b-0435-4fee-bf76-f2aef980b833  true     kworker1  Online  500GiB    100GiB     400GiB
+ pool-1-kworker2  aio:///dev/vdc?uuid=bb12ec7d-8fc3-4644-82cd-dee5b63fc8c5  true     kworker1  Online  500GiB    100GiB     400GiB
+ pool-1-kworker2  aio:///dev/vdb?uuid=f324edb7-1aca-41ec-954a-9614527f77e1  true     kworker2  Online  500GiB    100GiB     400GiB
 ```
 4. Get Pool by ID
 ```
 ❯ kubectl mayastor get pool mayastor-pool-1
```

Comment:
This one...

Reply:
I added the help on top where it shows the default namespace is mayastor, so don't need to keep repeating it.

Reply:
No, I mean the pool name. It's pool-1-kworker1

Reply:
Oh good catch, think that's all now